src/components/Radio/Radio.test.tsx

```diff
@@ -0,0 +1,13 @@
+import { shallow } from "enzyme";
+import React from "react";
+
+import Radio from "./Radio";
+
+describe("Radio ", () => {
+  it("renders", () => {
+    const wrapper = shallow(
+      <Radio label="Test radio" name="Radio test"></Radio>
+    );
+    expect(wrapper).toMatchSnapshot();
```

Comment:
We try to avoid using snapshots because it doesn't really test for a specific functionality. It also means that if you update anything in the component (or child components) this test will fail and require you to update and validate the new snapshot.

Might I propose updating the name of this test to something more specific like.... "Renders a radio input" and then your test can check for the input and any content you expect to be there directly.